application/jira/v0/config/tasks.json

```diff
@@ -0,0 +1,744 @@
+              ],
+              "instillEditOnNodeFields": [
+                "range"
+              ],
+              "instillFormat": "object",
+              "type": "object"
+            },
+            {
+              "properties": {
+                "range": {
+                  "const": "Issues of an epic",
+                  "type": "string"
+                },
+                "epic-key": {
+                  "title": "Epic Key",
+                  "description": "The Key of the epic",
+                  "instillShortDescription": "The Key of the epic",
+                  "instillUIOrder": 10,
+                  "instillFormat": "string",
+                  "instillAcceptFormats": [
+                    "string"
+                  ],
+                  "instillUpstreamTypes": [
+                    "value",
+                    "reference",
+                    "template"
+                  ],
+                  "type": "string"
+                }
+              },
+              "required": [
+                "range",
+                "epic-key"
+              ],
+              "instillEditOnNodeFields": [
+                "range",
+                "epic-key"
+              ],
+              "instillFormat": "object",
+              "type": "object"
+            },
+            {
+              "properties": {
+                "range": {
+                  "const": "Issues of a sprint",
+                  "type": "string"
+                },
+                "sprint-key": {
+                  "title": "Sprint Key",
+                  "description": "The Key of the sprint",
+                  "instillShortDescription": "The Key of the sprint",
+                  "instillUIOrder": 10,
+                  "instillFormat": "string",
+                  "instillAcceptFormats": [
+                    "string"
+                  ],
+                  "instillUpstreamTypes": [
+                    "value",
+                    "reference",
+                    "template"
+                  ],
+                  "type": "string"
+                }
+              },
+              "required": [
+                "range",
+                "sprint-key"
+              ],
+              "instillEditOnNodeFields": [
+                "range",
+                "sprint-key"
+              ],
+              "instillFormat": "object",
+              "type": "object"
+            },
+            {
+              "properties": {
+                "range": {
+                  "const": "In backlog only",
+                  "type": "string"
+                }
+              },
+              "required": [
+                "range"
+              ],
+              "instillEditOnNodeFields": [
+                "range"
+              ],
+              "instillFormat": "object",
+              "type": "object"
+            },
+            {
+              "properties": {
+                "range": {
+                  "const": "Issues without epic assigned",
```

Comment:
If we have a task for users to assign issues to an epic, users can use this to find issues that are not assigned to an epic.

Reply:
> do we need the options for standard issues?
After some research, we can achieve this by jql search to filter out epic and subtask. Since the middle layer (standard issues) can have multiple user-defined types, I will implement this function by jql query with "NOT type=Epic AND NOT type=subtask".

Reply:
Should we allow the user to directly input the JQL query?
It seems like there is no security issue for the JQL query.

[Advanced Search](https://support.atlassian.com/jira-software-cloud/docs/what-is-advanced-search-in-jira-cloud/)